scripts/repo-helper-fedora.sh

```diff
@@ -10,7 +10,11 @@ set -uex
 : "${REPOSITORY_NAME:=artifactory}"
 : "${archive:=}"
 if [ "$FVERSION" != "latest" ]; then
-    archive="-archive"
```

Comment:
what happens when 42 gets moved to archive?

Reply:
Most probably we will modify this script sooner when adding support for Fedora 43, 44.

But to be honest we should remove support for any `archive` version of Fedora. Archive means EOL.

Reply:
Fedora EOL is now calculated based on date retrieved from https://endoflife.date/api/fedora.json (via [artifactory](https://artifactory.daos.hpc.amslabs.hpecorp.net/ui/admin/repositories/remote/eol-proxy/))